firefly/main.py

```diff
@@ -99,6 +101,10 @@ def main():
 
     args = parse_args()
 
+    if args.version:
+        print("Firefly v{}".format(__version__))
```

Comment:
Please make it `Firefly version {}`. That would make it easier to extract version programmatically, if required.

Reply:
Okay.